src/pages/stablecoins.js

```diff
@@ -322,14 +321,25 @@ const StablecoinsPage = ({ data }) => {
     ;(async () => {
       try {
         // No option to filter by stablecoins, so fetching the top tokens by market cap
-        const data = await getData(
-          "https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&order=market_cap_desc&per_page=250&page=1&sparkline=false"
+
+        const ethereumData = await getData(
+          "https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&category=ethereum-ecosystem&order=market_cap_desc&per_page=100&page=1&sparkline=false"
+        )
+        const stablecoinData = await getData(
+          "https://api.coingecko.com/api/v3/coins/markets?vs_currency=usd&category=stablecoins&order=market_cap_desc&per_page=100&page=1&sparkline=false"
+        )
+
+        const filteredData = stablecoinData.filter(
```

Comment:
Perhaps we should add a comment to make clear what's happening here? Or perhaps a different variable name instead of `filteredData` to clarify, e.g. `ethereumStablecoinData`?

Reply:
Will give another pass on variable names, and some documentation. Guilty of not writing enough of that.